heudiconv/utils.py

```diff
@@ -38,7 +38,8 @@
     'accession_number',      # 21
     'patient_age',           # 22
     'patient_sex',           # 23
-    'date'                   # 24
+    'date',                  # 24
+    'uid'                    # 25
```

Comment:
I know that is a "duplicate" and so far we were not really consistent, but may be we should keep `series_` prefix as in `series_id`, `series_description`, to make it clear that it is a property of the series (khe khe -- seqinfo) and not of the entire exam/study (as in `study_description`) .  Would pair nice(r) with aforemented `series_id`.  I would even add `study_uid` (although I think it might be a part of `series_id` ATM...?)

Reply:
Agree. Will do.